internal/db/migrations/007_add_region.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE pubkey_resources
+ADD region text;
```

Comment:
We are just about to merge a huge PR that adds 007 and 008 migration files, can you rebase and rename this to 009?